includes/controllers/class.llms.controller.certificates.php

```diff
@@ -2,47 +2,83 @@
 /**
  * Certificate Forms
  *
- * @since   3.18.0
- * @version 3.24.0
+ * @since 3.18.0
+ * @version [version]
  */
 defined( 'ABSPATH' ) || exit;
 
 /**
  * @since 3.18.0
  * @since 3.35.0 Sanitize `$_POST` data.
+ * @since [version] Modify `llms_certificate` post type registration to allow certificate templates to be exported.
+ *                When exporting a certificate template, use the `post_author` for the certificate's WP User ID.
```

Comment:
since next version will be 3.37.4, I think we need to remove a space before "When".

Reply:
Your attention to detail is stupid.

Reply:
(with love)